ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -0,0 +1,121 @@
+import json
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import Optional
+
+from ddtrace._trace.span import Span
+from ddtrace.internal.logger import get_logger
+from ddtrace.llmobs._constants import INPUT_MESSAGES
+from ddtrace.llmobs._constants import METADATA
+from ddtrace.llmobs._constants import MODEL_NAME
+from ddtrace.llmobs._constants import OUTPUT_MESSAGES
+from ddtrace.llmobs._constants import SPAN_KIND
+
+from .base import BaseLLMIntegration
+
+
+log = get_logger(__name__)
+
+
+API_KEY = "anthropic.request.api_key"
+MODEL = "anthropic.request.model"
+TOTAL_COST = "anthropic.tokens.total_cost"
+TYPE = "anthropic.request.type"
+
+
+class AnthropicIntegration(BaseLLMIntegration):
+    _integration_name = "anthropic"
+
+    def llmobs_set_tags(
+        self,
+        span: Span,
+        input_messages: Optional[Dict[str, Any]] = None,
+        formatted_response: Optional[Dict[str, Any]] = None,
+        err: bool = False,
+    ) -> None:
+        """Extract prompt/response tags from a completion and set them as temporary "_ml_obs.*" tags."""
+        # if not self.llmobs_enabled:
+        #     return
+        parameters = {
+            "temperature": float(span.get_tag("anthropic.request.parameters.temperature") or 1.0),
+            "max_tokens": int(span.get_tag("anthropic.request.parameters.max_tokens") or 0),
+        }
+
+        input_messages = self._extract_input_message(input_messages)
+
+        span.set_tag_str(SPAN_KIND, "llm")
+        span.set_tag_str(MODEL_NAME, span.get_tag("anthropic.request.model") or "")
```

Comment:
In the OpenAI integration we had to save `anthropic.response.model` instead of `anthropic.request.model` because users could technically specify a generic model name i.e. `gpt-3.5-turbo` in the request, while the response would return the specific model name we care more about, i.e. `gpt-3.5-turbo-0901`.
Is this the same case for Anthropic? If so, let's keep that pattern.

Reply:
This is not the case. It will just throw an error if you try to do that with anthropic.

Reply:
So since the model name is consistent, should we only be setting `anthropic.request.model`?